ai21_tokenizer/jamba_instruct_tokenizer.py

```diff
@@ -63,9 +63,20 @@ def vocab_size(self) -> int:
         return self._tokenizer.get_vocab_size()
 
 
-class AsyncJambaInstructTokenizer(BaseJambaInstructTokenizer, BaseTokenizer):
-    def __init__(
-        self,
+class AsyncJambaInstructTokenizer(BaseJambaInstructTokenizer, AsyncBaseTokenizer):
+    _model_path: str
+    _tokenizer: Tokenizer = None
+    _cache_dir: PathLike = None
+
+    def __init__(self):
+        raise ValueError(
+            "Create object with context manager only.Use either AsyncJambaInstructTokenizer.create or "
```

Comment:
Error is wrong. You're saying to use only contextmanager and then giving the user options